mock/server/server.go

```diff
@@ -137,5 +137,8 @@ func FakeHandler() http.Handler {
 	e.POST("/authenticate/token", getAuthenticateFromToken)
 	e.GET("/validate-token", validateToken)
 
+	// mock endpoint for queue credentials
+	e.POST("/api/v1/queue/queue-registration", getQueueCreds)
```

Comment:
```suggestion
	e.POST("/api/v1/queue/register", getQueueCreds)
```

Avoids a stutter in the path. Just a suggestion though.

Reply:
How about register-queue? So that we can distinguish between token and queue?